TableSchemer/Scheme.swift

```diff
@@ -55,7 +56,7 @@ public protocol Scheme: class {
         - parameter   cell:          The UITableViewCell being created.
         - parameter   relativeIndex: The cell index from the start of the scheme being configured.
      */
-    func configureCell(cell: UITableViewCell, withRelativeIndex relativeIndex: Int)
+    func configureCell(_ cell: UITableViewCell, withRelativeIndex relativeIndex: Int)
```

Comment:
Do you think `configure(cell: UITableViewCell, withRelativeIndex relativeIndex: Int)` is more swift-3y?